server2/Arcadia.Assistant/Arcadia.Assistant.AppCenterBuilds.Contracts/Interfaces/IHelpSettings.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Arcadia.Assistant.AppCenterBuilds.Contracts
+{
+    public interface IHelpSettings
```

Comment:
why these interfaces in this library in the first place?

Reply:
Moved to Configuration declaration